actionpack/lib/action_controller/metal/conditional_get.rb

```diff
@@ -247,19 +240,14 @@ def stale?(object = nil, **freshness_kwargs)
     #
     # #### Options
     #
-    # `:public`
-    # :   If true, replaces the default `private` directive with the `public`
-    #     directive.
-    #
-    # `:must_revalidate`
-    # :   If true, adds the `must-revalidate` directive.
+    # `:public`: If true, replaces the default `private` directive with the `public`
+    #  directive.
     #
-    # `:stale_while_revalidate`
-    # :   Sets the value of the `stale-while-revalidate` directive.
+    # `:must_revalidate`: If true, adds the `must-revalidate` directive.
     #
-    # `:stale_if_error`
-    # :   Sets the value of the `stale-if-error` directive.
+    # `:stale_while_revalidate`: Sets the value of the `stale-while-revalidate` directive.
     #
+    # `:stale_if_error`: Sets the value of the `stale-if-error` directive.
```

Comment:
Why do these need to be changed? They aren't currently rendering as definition lists due to https://github.com/ruby/rdoc/commit/8f943bbba40c626015c1bd6494fd77f21a08552b but once RDoc is released they will.